src/main/scripts/chocolatey-package.bash

```diff
@@ -152,12 +159,20 @@ function main() {
         return 1
     fi
 
-    if ! update_resources $project_version target/rug-cli-${project_version}-bin.zip; then
+    # nuget doesn't support semver such as 1.0.0-m.2
+    # the .2 trail is not supported, so we create a nuget package without the dot
+    # see https://github.com/atomist/rug-cli/issues/189
+    local valid_nuget_project_version=$project_version
+    if [[ $project_version =~ ^[0-9]+\.[0-9]+\.[0-9]+(-(m|rc)\.[0-9]+)$ ]]; then
+        valid_nuget_project_version=$(echo $project_version | sed -rn 's/([0-9]+\.[0-9]+\.[0-9]+)(-m|-rc)\.([0-9]+)/\1\2\3/p')
```

Comment:
`-E` will work on more systems than `-r`, e.g., BSD systems like Mac OS X.

The `-n` option and `p` regex modifier must be removed.

You are not checking the exit status of the echo|sed command or ensuring valid_nuget_project_version ends up non-empty.

Lines 162--168 can probably be simplified to:

```sh
    # nuget does not support periods in pre-release sections of semantic versions
    local valid_nuget_project_version
    valid_nuget_project_version=$(echo $project_version | sed -E 's/([^-]+)-(m|rc)\.([0-9]+)/\1-\2\3/')
    if [[ $? -ne 0 || ! $valid_nuget_project_version ]]; then
        err "failed to create valid nuget version from $project_version: $valid_nuget_project_version"
        return 1
    fi
```